### ISSUE TITLE
Still trying to export table even though INCLUDE_TABLES = ""

### ISSUE THREAD

**Issue by dylanadd** [timabell/msaccess-vcs-integration#12]
Hi,

The title just about says it all. The process fails when it gets to the export table def function. It gets even weirder because the script ends up calling the export linked table function. Any ideas on why it would do this?

**Comment by dylanadd** [comment]
It throws a "object or method not found" error.

**Comment by jwbrookes** [comment]
Table defs are always exported. The `INCLUDE_TABLES` variable controls the export of table data.

**Comment by jwbrookes** [comment]
What is the error number?

What version of Access are you using?

**Comment by dylanadd** [comment]
Using access 2010. 
You were right about it not exporting the actual table, just the definition.

I resolved the issue by changing some of the variable declarations in the VCS_ImportExport script and the VCS_Table script from specific object types (such as Index, TableDef, Field, etc.) to the generic "Object" type. For whatever reason, not all of the objects properties were initializing with some of the tables, but when I changed the type to "Object" it worked like a charm.
